packages/core/src/model/model.ts

```diff
@@ -46,6 +41,34 @@ export class ChartModel {
 		this.services = services
 	}
 
+	formatTable(headingLabels, tableData) {
```

Comment:
can we pls update this function to just take an object instead of 2 parameters

e.g. `formatTable({ headers: [...], cells: [...] })` this way it's easier to read